packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -382,14 +381,12 @@ export default class CodeSubmode extends Component<Signature> {
     return this.card;
   }
 
-  @action
-  private initializeMonacoCursorPosition() {
+  private get initialMonacoCursorPosition() {
```

Comment:
we should probably use `@cached` so that it doesnt make a new Position instance every time this is called--but reuses the same one it made before if none of the vars that it depends on haven;t changed